app/customer_routes.py

```diff
@@ -0,0 +1,129 @@
+from app import db
+from flask import Blueprint, jsonify, request
+from app.models.customer import Customer
+# from sqlalchemy_paginator import Paginator
+from dotenv import load_dotenv
+load_dotenv()
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+
+#HELPER FUNCTION
+def find_specific_key_error(request_body):
+    if "postal_code" not in request_body.keys():
+            response = {
+                "details": "Request body must include postal_code."
+            }
+    elif "name" not in request_body.keys():
+        response = {
+            "details": "Request body must include name."
+        }
+    elif "phone" not in request_body.keys():
+        response = {
+            "details": "Request body must include phone."
+        }
```

Comment:
keys are the default iterator for dictionaries so we can actually omit the `keys()` method and this conditional will work the same way.

```suggestion
    if "postal_code" not in request_body:
            response = {
                "details": "Request body must include postal_code."
            }
    elif "name" not in request_body:
        response = {
            "details": "Request body must include name."
        }
    elif "phone" not in request_body:
        response = {
            "details": "Request body must include phone."
        }
```